File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_5.py
salario = float(input('Digite o salário do funcionário:'))

if salario > 0:
    print('Tabela de cargos:\n'
        '1 - Programador de Sistemas\n'
        '2 - Analista de Sitemas\n'
        '3 - Analista de Banco de Dados\n')
    cargo = int(input('Digite o cargo do funcionário:'))

    match cargo:
        case 1:
            print(f'novo salário:{salario+(salario*(30/100))}')
        case 2:
            print(f'novo salário:{salario+(salario*(20/100))}')
        case 3:
            print(f'novo salário:{salario+(salario*(15/100))}')
        case _:
            print('Cargo inválido')
else:
    print('Salário deve ser maior que zero!')

File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-5.js
const prompt = require('prompt-sync')()

let salarioFinal

const salario = Number(prompt('Digite o salário atual do funcionário: '))
console.log(`
Cargo| COD 1 | Programador de Sistemas
Cargo| COD 2 | Analista de Sistemas
Cargo| COD 3 | Analista de Banco de Dados`)

if (salario > 0){
    const codigo = Number(prompt('Digite o código do cargo: '))
    switch(codigo){
        case 1:
            salarioFinal = salario * (1 + 30/100)
            break

        case 2:
            salarioFinal = salario * (1 + 20/100)
            break

        case 3:
            salarioFinal = salario * (1 + 15/100)
            break

        default:
            console.log('Código inválido')
            break
    }
}else{
    console.log('Salário deve ser positivo!')
}

salarioFinal != undefined? console.log(`Salário Final:${salarioFinal}`) : console.log(``)